tests/stub/configuration_hints/test_connection_recv_timeout_seconds.py

```diff
@@ -90,12 +96,42 @@ def test_timeout_unmanaged_tx(self):
         res.next()
         tx.commit()
         self._server.done()
-        self._assert_is_timout_exception(exc.exception)
+        self._assert_is_timeout_exception(exc.exception)
         self.assertEqual(self._server.count_responses("<ACCEPT>"), 2)
         self.assertEqual(self._server.count_responses("<HANGUP>"), 2)
         self.assertEqual(self._server.count_requests('RUN "timeout"'), 1)
         self.assertEqual(self._server.count_requests('RUN "in time"'), 1)
 
+    @driver_feature(types.Feature.CONF_HINT_CON_RECV_TIMEOUT)
+    def test_timeout_unmanaged_tx_should_fail_subsequent_usage_after_timeout(
+            self):
+        self._start_server("1_second_exceeds_tx.script")
+        tx = self._session.beginTransaction()
+        with self.assertRaises(types.DriverError) as exc:
+            result = tx.run("timeout")
+            # TODO It will be removed as soon as JS Driver
+            # has async iterator api
+            if get_driver_name() in ['javascript']:
+                result.next()
+
+        try:
+            result = tx.run("in time")
+            if get_driver_name() in ['javascript']:
+                result.next()
+        except types.DriverError as e:
+            self._assert_is_client_exception(e)
```

Comment:
This does not assert that an exception is raised. If no error occurs, the test still passes.

```python
with self.assertRaises(types.DriverError) as e:
    run_stuff(something)
self._assert_is_client_exception(e.exception)
```

Reply:
Good point, I will update it. Thank you 👍